pennylane/transforms/qcut.py

```diff
@@ -2039,18 +2205,23 @@ class CutStrategy:
         min_free_wires (int): Number of wires for the smallest available device, or, equivalently,
             the smallest max fragment-wire-size that the partitioning is allowed to explore.
             When provided, this parameter will be used to derive an upper-bound to the range of
-            explored number of fragments.  Optional, defaults to ``max_free_wires``.
+            explored number of fragments.  Optional, defaults to 2 which corresponds to attempting
+            the most granular partitioning of max 2-wire fragments.
         num_fragments_probed (Union[int, Sequence[int]]): Single, or 2-Sequence of, number(s)
             specifying the potential (range of) number of fragments for the partitioner to attempt.
             Optional, defaults to probing all valid strategies derivable from the circuit and
-            devices.
+            devices. When provided, has precedence over all other arguments affecting partitioning
+            exploration, such as ``max_free_wires``, ``min_free_wires``, or ``exhausive``.
```

Comment:
```suggestion
            exploration, such as ``max_free_wires``, ``min_free_wires``, or ``exhaustive``.
```
May need to do a find and replace for `exhausive` -> `exhaustive`